From c67466803b0885e28d3a7c2179433489659d258b Mon Sep 17 00:00:00 2001
From: Vishal Sagar <vishal.sagar@xilinx.com>
Date: Wed, 26 Sep 2018 11:02:16 +0530
Subject: [PATCH 0419/1852] documentation: dt: vtpg: Add support for max width
 and height

commit eb9537f4ae0738d1ad66e7f564d5cf595956a272 from
https://github.com/Xilinx/linux-xlnx.git

Adds two optional parameters xlnx,max-width and xlnx,max-height which
determine the max size of frame the IP can process.

Signed-off-by: Vishal Sagar <vishal.sagar@xilinx.com>
Reviewed-by: Hyun Kwon <hyun.kwon@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 Documentation/devicetree/bindings/media/xilinx/xlnx,v-tpg.txt | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/Documentation/devicetree/bindings/media/xilinx/xlnx,v-tpg.txt b/Documentation/devicetree/bindings/media/xilinx/xlnx,v-tpg.txt
index f12c244c26a5..d125a547abe8 100644
--- a/Documentation/devicetree/bindings/media/xilinx/xlnx,v-tpg.txt
+++ b/Documentation/devicetree/bindings/media/xilinx/xlnx,v-tpg.txt
@@ -33,6 +33,10 @@ Optional properties:
 - reset-gpios: Specifier for a GPIO that assert TPG (AP_RST_N) reset.
   This property is mandatory for TPG v7.0.
 
+- xlnx,max-height: Maximum number of lines.
+
+- xlnx,max-width: Maximum number of pixels in a line.
+
 The xlnx,vtc and timing-gpios properties are mandatory when the TPG is
 synthesized with two ports and forbidden when synthesized with one port.
 
-- 
2.31.1

